ScrutinyPlot/plot3periods.py

```diff
@@ -0,0 +1,59 @@
+# Takes three file names, for full period, half of period, and fraction of period
+
+import ROOT
```

Comment:
I would prefer to stay away from ROOT for such simple task. Just brining this dependency make it hard to run a script in non-CERN/non-ROOT environment. The de-factor standard in python for making plots is matplotlib. I would appreciate if you'll redo this script using matplotlib. An additional argument, that we may run such script in a cron which will require to setting up a special environment for ROOT python package. The later itself requires to install ROOT.
 
Or, you can use histogrammar package (http://histogrammar.org/) to do plotting. It is developed by DIANA-HEP group and much more intuitive and easy to use (in python sense) than ROOT.

Reply:
I don't know matplotlib or histogrammar. After I learn how to use them, I might be able to translate this ROOT program, but I don't know when I will have time to study these packages. Maybe someone else will be able to perform the translation.